app/models/plan.rb

```diff
@@ -24,6 +26,8 @@ class Plan < ApplicationRecord
   validates :code, presence: true, uniqueness: { scope: :organization_id }
   validates :amount_currency, inclusion: { in: currency_list }
 
+  scope :default, -> { where(overridden_plan_id: nil) }
```

Comment:
`default_scope` instead?

Reply:
I can switch to default scope but I personally don't like it since it mask behaviour :/

Reply:
I tend to agree 😅 

But I think we can use a different name than `default` then, something more explicit?

Reply:
`root`? 😃

Reply:
`parent` makes more sense I think.

`default` could means many many things :smile:

Reply:
what about `base` ?

Reply:
switched to `base`

Reply:
`not_overriden`, explicit 😅